core/src/main/java/org/apache/carbondata/events/OperationContext.java

```diff
@@ -41,4 +41,12 @@ public void setProperty(String key, Object value) {
   public Object getProperty(String key) {
     return this.operationProperties.get(key);
   }
+
+  public Object getOrElse(String key, Object defaultValue) {
```

Comment:
This method is not getting used anywhere. If not required remove this method

Reply:
removed